x-pack/plugins/lens/public/plugin.ts

```diff
@@ -160,6 +161,8 @@ export class LensPlugin {
   start(core: CoreStart, startDependencies: LensPluginStartDependencies) {
     this.attributeService = getLensAttributeService(core, startDependencies);
     this.createEditorFrame = this.editorFrameService.start(core, startDependencies).createInstance;
+    // unregisters the OSS alias
+    startDependencies.visualizations.unRegisterAlias(OSS_PLUGIN_ID);
```

Comment:
I added to the start of the plugin because on the setup I fell into race conditions.